pkg/util/uri.go

```diff
@@ -15,7 +15,11 @@ package util
 
 import (
 	"net"
+	"net/url"
 	"strings"
+
+	"github.com/ngaut/log"
```

Comment:
Maybe it's better to use `github.com/pingcap/log` to keep consistent with other files.

Reply:
good catch